docs/pages/styled-jsx/+Page.mdx

```diff
@@ -1,13 +1,20 @@
 import { Example, UiFrameworkExtension, CommunityNote } from '../../components'
 import { Link } from '@brillout/docpress'
 
-<CommunityNote tool="styled-jsx" url="https://github.com/vercel/styled-jsx" hasExtension={false} />
+<CommunityNote tool="styled-jsx" url="https://github.com/vercel/styled-jsx" hasExtension="react" />
```

Comment:
Let's add backticks. Its official name is with backticks (let's always look at how tool authors write their tool).

Reply:
Where should we add the backticks ?

Reply:
to styled-jsx

Reply:
Make the argument accept `JSX.Element`. `Intervention+=2`.

Reply:
Well, it's not from an argument but `pageContext.pageTitle`. I guess we can add an optional prop `toolTitle?: string | JSX.Element` for `<CommunityNote>`

Reply:
Ah, I see. How about we change it in `headingsDetached.ts` then and that's it.

Reply:
It doesn't work; the backticks are rendered literally. That's why I removed them in the first place

![Screenshot 2024-12-17 at 22 41 17](https://github.com/user-attachments/assets/5052e5c8-8e97-468d-9319-44df6ee0d0f4)

Reply:
:+1: Alright, let's stick to styled-jsx without backticks then.